### PR TITLE
change smoketest to use AWS+ubuntu

### PR DESCRIPTION
**Pull request by PratikDeoghare**
**What this PR does / why we need it**:
Current dualstack smoketest tests all the OS on Azure. 
Testing all the OS is expensive. 
Azure cleanup is leaky. 
We switch to testing only Ubuntu on AWS for smoketest because it runs frequently.  

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: pkg/test/dualstack/dualstack_test.go [hunk 1]
```diff
@@ -54,6 +55,7 @@ var (
 func init() {
 	flag.StringVar(&userconfig, "userconfig", "", "path to kubeconfig of usercluster")
 	flag.StringVar(&ipFamily, "ipFamily", "IPv4", "IP family")
+	flag.StringVar(&osNames, "os", "", "Comma separated list of operating systems to test ubuntu,flatcar")
```
Comment (rastislavs) [inline pkg/test/dualstack/dualstack_test.go]: maybe missing `e.g.` in front of `ubuntu,flatcar`?

**Comment by PratikDeoghare** [comment]
@xrstf Can this merge now? :thinking:

**Comment by xrstf** [comment]
Ohh boy, sorry, I forgot this :facepalm: 

/unhold

**Pull request merged**